Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Column Name	Description	Ontology identifier	Ontology term name	Ontology Description
+Specimen ID	Unique identifier for this specimen	OBI:0100051	specimen	A material entity that has the specimen role.
+Parent specimen ID	Identifier for this specimen’s parent specimen (optional)	OBI:0100051	specimen	A material entity that has the specimen role.
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
+Specimen type	Type of specimen (excised tissue block, slice, cell, etc.)	AIO:specimen type name	specimen type name	A textual entity that denotes the type of a specimen.
+Hemisphere	Hemisphere that this specimen came from (left, right, both, unknown)	PATO:0000140	position	A spatial quality inhering in a bearer by virtue of the bearer's spatial location relative to other objects in the vicinity.
+Anatomical structure	Specimens location within anatomical structures from brain atlas	PATO:0000141	structure	A morphology quality inhering in a bearer by virtue of the bearer's relative position, shape, arrangements and connectivity of an organism's various parts; the pattern underlying its form.
+Specimen disease condition	Description of specimen’s disease state (healthy vs diseased tissue)	OBI:0000278	disease stage	A part of an occurrence of a disease process which is associated with position in the normal progression of the disease
+Subject organism ID	ID for the subject organism that this specimen came from	OBI:0100026	organism	A material entity that is an individual living system, such as animal, plant, bacteria or virus, that is capable of replicating or reproducing, growth and maintenance in the right environment. An organism may be unicellular or made up, like humans, of many billions of cells divided into specialized tissues and organs.
+Organism age time measurement	Age of organism at the time of initial specimen creation	OBI:0001167	age measurement datum	A time measurement datum that is the result of measurement of age of an organism
```

Comment:
Depending on how this is interpreted there might be issues. If samples are taken from the same subject at multiple times this won't work because the subject will have multiple ages. If each one of these sheets is treated as distinct and has a deposition date then it will work (ish) however, I think this probably needs to have a sample collection date so that the subject age makes sense (subject birth date is more generalized here, but we can't always get it).

Reply:
All the fields relating to the subject organism are intended to be snapshots of the organism at the time of specimen creation. Therefore, we would expect something like organism age to possibly be different between specimens. The subject organism ID is the only thing that is guaranteed to be the same for all specimens derived from one individual organism.